src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import cn from 'classnames';
+import { FilterOptions } from '../../types/FilterOptions';
+
+type Props = {
+  filterOption: FilterOptions;
+  onChangeFilterOption: React.Dispatch<React.SetStateAction<FilterOptions>>;
+  activeTodosCount: number;
+};
+
+export const TodoFooter: React.FC<Props> = ({
+  filterOption,
+  onChangeFilterOption,
+  activeTodosCount,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${activeTodosCount} items left`}
+      </span>
+
+      <nav className="filter">
+        <a
+          href="#/"
+          className={cn(
+            'filter__link',
+            { selected: filterOption === FilterOptions.All },
+          )}
+          onClick={() => onChangeFilterOption(FilterOptions.All)}
```

Comment:
Suggest to create one handler for all filter buttons and just call with proper value in jsx
```
const handleFilter = (filterField: FilterOptions) => () => onChangeFilterOption(filterField);
```

```suggestion
          onClick={handleFilter(FilterOptions.All)}
```